framework/test/integration/testing/block_processing_env.spec.ts

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
i think we forgot to create issue for upgrading the integration tests to use this new function. We should test it if all the interface are usable at least for us

Reply:
https://github.com/LiskHQ/lisk-sdk/issues/6203